modules/jooby-validation/src/main/java/io/jooby/validation/MvcValidator.java

```diff
@@ -0,0 +1,6 @@
+package io.jooby.validation;
+
+public interface MvcValidator<E extends Throwable> {
+
+    void validate(Object bean) throws E;
```

Comment:
well not sure if we need a generic for exception, RuntimeExceptoion or Exception will be OK